.github/workflows/cd.yaml

```diff
@@ -57,58 +43,57 @@ jobs:
       - run: |
           set -x
           NAMESPACE=ci-${{ github.run_number }}
+          SPACE=files
```

Comment:
Nit: Using all capitals for these makes them look like environment variables, so maybe use lowercase.